### PR TITLE
feat Product page

### PR DESCRIPTION
**Pull request by AmishMishra11**
added product page

Filters include
- Clear filter
- Price range
- Category
- Rating
- Sory By

Screenshot (21)

### CODE REVIEW

File: product/product.css [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+  align-items: center;
+  cursor: pointer;
+  position: relative;
+}
+
+.checkbox > span {
+  padding: var(--gap-smallest);
+}
+
+.checkbox > input {
```
Comment (Hardiegogo) [inline product/product.css]: Use a class instead of using child selector.

File: product/text.css [hunk 2]
```diff
@@ -0,0 +1,89 @@
+input[type="range"] {
```
Comment (Hardiegogo) [inline product/text.css]: Remove unused code.

**Review by Hardiegogo — COMMENTED** [review]
Look at some suggested changes below.
Everything else looks good.

File: authentication/login.css [hunk 3]
```diff
@@ -66,12 +66,61 @@
...
+.checkbox > input:active {
+  border: 2px solid black;
+}
+/* .container-reminder input {
   border-radius: 5px;
   margin-right: 5px;
   height: 1.2rem;
   width: 1.2rem;
-}
+} */
```
Comment (KanekarSnehal) [inline authentication/login.css]: remove comments

**Review by KanekarSnehal — APPROVED** [review]
Well structured.

File: product/text.css [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+  width: 15px;
+  border-radius: 5px;
+  background: #ffffff;
+  cursor: pointer;
+}
+input[type="range"]:focus::-ms-fill-lower {
+  background: #3071a9;
+}
+input[type="range"]:focus::-ms-fill-upper {
+  background: #3071a9;
```
Comment (nerdyspook) [inline product/text.css]: You can declare global variables for these colors as it is repeated a few times.

**Review by nerdyspook — APPROVED** [review]

File: product/text.css (hunk 4 above)
Comment (AmishMishra11) [reply product/text.css]: that was unnecessary code ive removed it

File: authentication/login.css (hunk 3 above)
Comment (AmishMishra11) [reply authentication/login.css]: yes thank you

File: product/text.css (hunk 2 above)
Comment (AmishMishra11) [reply product/text.css]: thank you for pointing it out

File: product/product.css (hunk 1 above)
Comment (AmishMishra11) [reply product/product.css]: yes ive fixed that

**Pull request merged**